lib/new_playground.dart

```diff
@@ -65,10 +65,11 @@ class Playground implements GistContainer, GistController {
   MDCButton runButton;
   MDCButton editorConsoleTab;
   MDCButton editorDocsTab;
+  MDCButton closePanelButton;
   DElement editorPanelFooter;
   MDCMenu samplesMenu;
   Dialog dialog;
-  DContentEditable titleEditable;
+  DElement titleEditable;
```

Comment:
nit: Consider a different name if this isn't user-editable.

Reply:
fixed